RELEASE/scripts/sl_ascend.ash

```diff
@@ -12599,6 +12611,10 @@ boolean L9_oilPeak()
 	if(get_property("lastAdventure") == "Unimpressed with Pressure")
 	{
 		set_property("oilPeakProgress", 0.0);
+		// Brute Force grouping with tavern (if not done) to maximize tangles while we have a high ML.
+		print("Checking to see if we should do the tavern while we are running high ML.", "green");
+		forceTavern = true;
+		L3_tavern();
```

Comment:
Don't directly call `L3_tavern` from here, once you switch `forceTavern` in to a property it should just automatically happen if it can.